src/sonic-device-data/tests/platformJson_checker

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+try:
+    import re
+    import sys
+    import glob
+    import json
+except ImportError as e:
+    raise ImportError (str(e) + "- required module not found")
+try:
+  basestring
+except NameError:
+  basestring = str
```

Comment:
Inconsistent indenting. Please unify all indents for four spaces.

Reply:
**#Modified**